mii/config.py

```diff
@@ -84,6 +86,12 @@ def checkpoint_dict_valid(cls, value):
                 raise ValueError(f"Missing key={k} in checkpoint_dict")
         return value
 
+    @validator('enable_load_balancing')
+    def enable_load_balancing_valid(cls, field_value, values):
+        if values["enable_restful_api"]:
+            field_value = True
+        return field_value
+
```

Comment:
```suggestion
    @root_validator
    def auto_enable_load_balancing(cls, values):
        if values["enable_restful_api"]:
            values["enable_load_balancing"] = True
        return values
```
If we make this a root validator, the ordering of the fields won't matter. With the current validator, if a future PR places the `enable_load_balancing` above `enable_restful_api` we would get the wrong behavior.

Reply:
Thank you for the proposal! I'm learning pydantic.